The dbbackup.bat file does not run correctly on Windows.

**Describe The Bug**
When I try to run a backup on Windows, the script fails with an error message

```
Error: Could not find or load main class org.h2.tools.Script
Caused by: java.lang.ClassNotFoundException: org.h2.tools.Script
```

The issue appears to be the classpath creation. When the command is build manually then the tool runs correctly.

**To Reproduce The Bug**
Steps to reproduce the behavior. Please be as complete as possible and describe the exact steps and values used to cause the error:
1. Run the dbbackup.bat file on Windows
1. An error message is output.

**Expected Behavior**
1. The backup should complete successfully.

**ComiXed Runtime Environment**

 - [X] Windows
 - [ ] Linux (any distro)
 - [ ] MacOS
 - [ ] Docker (if so, please select the host OS)

**Browser (If A Web Issue)**

 - [ ] Chrome
 - [ ] Firefox
 - [ ] Safari
 - [ ] Other (if so, please provide a name)

--- a/comixed-release/src/main/assembly/scripts/dbbackup.bat
+++ b/comixed-release/src/main/assembly/scripts/dbbackup.bat
@@ -22,6 +22,8 @@ CD /d %~dp0
 SET DATABASE=~/.comixed/comixed
 SET DBUSER=sa
 
+FOR %%f IN (..\lib\h2*jar) DO SET H2_JAR=%%f
+
 :process_command_line
 IF "%~1" == "" GOTO end_process_command_line
 SET PARAM=%~1
@@ -77,7 +79,7 @@ IF "%DBPWRD%" == "" GOTO skip_set_password
 SET OPTIONS=%OPTIONS% -password %DBPWRD%
 :skip_set_password
 
-java -cp %LIBDIR%/h2*jar org.h2.tools.Script -url jdbc:h2:file:%DATABASE% %OPTIONS% -script comixed-backup-2.0.0-0.dev.zip -options compression zip
+java -cp %H2_JAR% org.h2.tools.Script -url jdbc:h2:file:%DATABASE% %OPTIONS% -script comixed-backup-2.0.0-0.dev.zip -options compression zip
 
 :exit_script
 ENDLOCAL

--- a/comixed-release/src/main/assembly/scripts/dbrestore.bat
+++ b/comixed-release/src/main/assembly/scripts/dbrestore.bat
@@ -22,6 +22,8 @@ CD /d %~dp0
 SET DATABASE=~/.comixed/comixed
 SET DBUSER=sa
 
+FOR %%f IN (..\lib\h2*jar) DO SET H2_JAR=%%f
+
 :process_command_line
 IF "%~1" == "" GOTO end_process_command_line
 SET PARAM=%~1
@@ -83,7 +85,7 @@ IF "%DBPWRD%" == "" GOTO skip_set_password
 SET OPTIONS=%OPTIONS% -password %DBPWRD%
 :skip_set_password
 
-java -cp %LIBDIR%/h2*jar org.h2.tools.RunScript -url jdbc:h2:file:%DATABASE% %OPTIONS% %BACKUP_FILENAME% -options compression zip
+java -cp %H2_JAR% org.h2.tools.RunScript -url jdbc:h2:file:%DATABASE% %OPTIONS% %BACKUP_FILENAME% -options compression zip
 
 :exit_script
 ENDLOCAL

